CONFIGURATION.md

```diff
@@ -156,6 +156,48 @@ SolarWindsAPM::Config[:transaction_settings] = [
 ]
 ```
 
+### Tag Trace Context with Query
```

Comment:
This is a great addition! Do you mind if i commit a change with some tweaks?

Reply:
yes, please do so